### PR TITLE
Image Service - Single Image

### PR DESCRIPTION
**Pull request by mikeapp**
Recipe #5

### CODE REVIEW

**Comment by mikeapp** [comment]
#5 lists two 2.1 fixtures - one with a simple image service, the other contains additional properties for the service.  I'm unsure how we want to handle this - split the issue or have two examples in recipe, with additional explanation.

File: cookbook/0005-image-service/index.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  ],
+  "id": "https://example.org/iiif/book5/manifest",
+  "type": "Manifest",
+  "label": { "en": [ "Simple Manifest - Image with IIIF Image API Service" ] },
+  "items": [
+    {
+      "id": "https://example.org/iiif/book5/canvas/p1",
+      "type": "Canvas",
+      "height": 1800,
+      "width": 1200,
```
Comment (stephenwf) [inline cookbook/0005-image-service/index.md]: Should we add a label to the canvas (marked as recommended in the specification)

**Comment by mikeapp** [comment]
Probably.  I copied the example from mvm-image, which also lacks a label - but "minimum viable" is not equivalent to best practice.

File: cookbook/0005-image-service/index.md (hunk 1 above)
Comment (mikeapp) [reply cookbook/0005-image-service/index.md]: Probably. I copied the example from mvm-image, which also lacks a label - but "minimum viable" is not equivalent to best practice.

Comment (stephenwf) [reply cookbook/0005-image-service/index.md]: That makes sense 👍

**Comment by mikeapp** [comment]
Per discussion on IIIF/api#1769 expand on annotation `id` needing to reference an image but not necessarily derived from a Image API service.

**Comment by mikeapp** [comment]
Also duplicated by @tomcrane  in themed branch

**Pull request closed without merging**